integration/tcp_test.go

```diff
@@ -346,3 +326,54 @@ func guessWhoTLSMaxVersion(addr, serverName string, tlsCall bool, tlsMaxVersion
 
 	return string(out[:n]), nil
 }
+
+// guessWhoTLSPassthrough guesses service identity and ensure that the certificate is valid for the given server name.
```

Comment:
// guessWhoTLSPassthrough guesses service identity and ensures that the
// certificate is valid for the given server name.